msteams-platform/messaging-extensions/what-are-messaging-extensions.md

```diff
@@ -66,6 +66,158 @@ The following images display link unfurling feature when a link is pasted in mes
 
 ![link unfurling](../assets/images/messaging-extension/link-unfurl.gif)
 
+## Code snippets
+
+The following code provides an example of action based for messaging extensions:
+
+# [Node.js](#tab/nodejs)
```

Comment:
To keep it consistent with other doc pages, could you please change the sequence to bring C# tab first? 
<img width="225" alt="image" src="https://user-images.githubusercontent.com/31851992/150472296-93847ce8-2ce0-466d-827e-41030b41742f.png">

Reply:
made changes as advised.